prow/config/config.go

```diff
@@ -1600,6 +1617,14 @@ func parseProwConfig(c *Config) error {
 		}
 	}
 
+	if c.Deck.Spyglass.GCSBrowserPrefixes == nil {
+		c.Deck.Spyglass.GCSBrowserPrefixes = make(map[string]string)
+	}
+
+	if _, exists := c.Deck.Spyglass.GCSBrowserPrefixes["*"]; !exists {
```

Comment:
Needs to error if both `c.Deck.Spyglass.GCSBrowserPrefix` and `c.Deck.Spyglass.GCSBrowserPrefixes["*"]` exist. Also, please deprecate the former (`ANNOUNCEMENTS.md` note, warning during startup)